src/routes/api.js

```diff
@@ -1,9 +1,23 @@
 import React from "react"
 import { ThreeColumnLayout } from "../components/three-column-layout"
+import useApiDocs from "../hooks/use-api-docs"
```

Comment:
named export vs. default? does it matter?

also, does this need to be a hook? It doesn't have any run-time state right?

Reply:
👍 We should used named, probably refactor the current default exports as we come across em.

Needs to be a hook since it pulls data from the mesh.